src/transformers/quantizers/quantizer_bnb_4bit.py

```diff
@@ -64,16 +65,25 @@ def __init__(self, quantization_config, **kwargs):
             self.modules_to_not_convert = self.quantization_config.llm_int8_skip_modules
 
     def validate_environment(self, *args, **kwargs):
-        if not torch.cuda.is_available():
-            raise RuntimeError("No GPU found. A GPU is needed for quantization.")
-        if not is_accelerate_available():
+        if not (is_accelerate_available() and is_bitsandbytes_available()):
             raise ImportError(
                 f"Using `bitsandbytes` 4-bit quantization requires Accelerate: `pip install 'accelerate>={ACCELERATE_MIN_VERSION}'`"
             )
         if not is_bitsandbytes_available():
             raise ImportError(
                 "Using `bitsandbytes` 4-bit quantization requires the latest version of bitsandbytes: `pip install -U bitsandbytes`"
             )
+        import bitsandbytes as bnb
+
+        bnb_is_multibackend_enabled = "multi_backend" in getattr(bnb, "features", set())
```

Comment:
Nice ! Just a small suggestion
```suggestion
        bnb_multibackend_is_enabled = "multi_backend" in getattr(bnb, "features", set())
```